demo-extension/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+	"manifest_version": 2,
+	"version": "1",
+	"name": "webext-patterns",
+	"background": {
+		"scripts": ["background.js"]
+	},
+	"content_scripts": [
+		{
+			"matches": ["*://*/*"],
+			"include_globs":["*.com/404/*"],
```

Comment:
1. Run `web-ext run`
2. Make changes to this glob
3. See whether content has been injected to the tab you're looking at